devicon.json

```
@@ -5776,6 +5776,22 @@
         },
         "color": "#294172", 
         "aliases": [] 
-    }
-
+    },
+    {
+    "name": "vitejs",
+    "tags": ["react", "compiler", "esbuild"],
+    "versions": {
+      "svg": [
+	      "original", 
+	      "original-wordmark",
+      ],
+      "font": ["original"]
+    },
+    "color": "#4FBFFF",
+    "aliases": [
+      {
+        "base": "original"
```

You need to also add an alias here.
```suggestion
        "base": "original"
        "alias": "plain"
```

I suggest you take a look at [Example-of-Submitting-Icons-to-the-Repository](https://github.com/devicons/devicon/wiki/Example-of-Submitting-Icons-to-the-Repository) and [Updating the `devicon.json`](https://github.com/devicons/devicon/wiki/Updating-the-%60devicon.json%60) to see why it should be like this.